### PR TITLE
org_text adjustments

### PR DESCRIPTION
**Pull request by to-sta**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
I adjusted the org API to create the OrganizationText for POST requests. 

ToDo:
* Toast needs to be added on the frontend, for bad requests.

### Related issue

- #915

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for this, @to-sta!

Are you planning on adding Toast yourself, or should I check this? And let me know what changes should generally happen in the [organization store](https://github.com/activist-org/activist/blob/main/frontend/stores/organization.ts) :)

**Comment by to-sta** [comment]
@andrewtavis I will give it a try, with [vue-sonner](https://github.com/xiaoluoboding/vue-sonner). 

We don't need the OrganizationText POST request anymore in the store. I can remove it myself while working on the toast 🤗

**Comment by andrewtavis** [comment]
Sounds good! Thanks, @to-sta!

**Comment by to-sta** [comment]
Removed OrganizationText request and added in the vue sonner toast in the bottom left.

**Comment by andrewtavis** [comment]
Nice, @to-sta! Should I look into the conflicts from here? Let me know :)

**Comment by to-sta** [comment]
> Nice, @to-sta! Should I look into the conflicts from here? Let me know :)

Yeah, that would be great. 

But I think instead of resolving 150 conflicts for versions, it would be more effective to re-compile the dependencies. You could remove the `yarn.lock` before merging it in and re-compile afterwards.

**Comment by andrewtavis** [comment]
Ah ya, 100% @to-sta :) I don't deal with `yarn.lock` conflicts. Always just swap in the most recent version of things and recompile 😊

File: frontend/pages/organizations/create.vue [hunk 1]
```diff
@@ -117,6 +118,8 @@
 
 <script setup lang="ts">
 import type { OrganizationCreateFormData } from "~/types/entities/organization";
+import {Toaster, toast } from 'vue-sonner'
+import BtnAction from "~/components/btn/action/BtnAction.vue";
```
Comment (andrewtavis) [inline frontend/pages/organizations/create.vue:122]: Quick note that we don't need to import the component in Nuxt, but then I think this was likely an auto import? :)

**Comment by andrewtavis** [comment]
Quick note here, @to-sta: I'm not able to navigate to the organization about page post creation with the current state of things. I can take a look at this a bit later when I fix the `yarn.lock` file :)

File: frontend/pages/organizations/create.vue (hunk 1 above)
Comment (to-sta) [reply frontend/pages/organizations/create.vue]: Yes, I didn't even realize that it was added to the imports 😄.

**Comment by to-sta** [comment]
> Quick note here, @to-sta: I'm not able to navigate to the organization about page post creation with the current state of things. I can take a look at this a bit later when I fix the `yarn.lock` file :)

What was the response? I just checked if I removed the redirect, but it is still in their and it works fine on my end.  It could be the db volume since the models have changed.

**Comment by andrewtavis** [comment]
I'll give it another shot on Sunday and let you know :) Thanks for the note that it's working on your end!

**Pull request merged**